modules/services/backup/restic.nix

```diff
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  options = {
```

Comment:
Every module needs a `enable` option. See the other module on how to make one with `mkEnableOption`